h/templates/includes/annotation_card.html.jinja2

```diff
@@ -1,47 +1,53 @@
-{% macro annotation_card(annotation, request, incontext_link, group=none) -%}
-
+{% macro annotation_card(presented_annotation, request, tag_link) -%}
```

Comment:
It would be helpful here to have some documentation about the expected types of these arguments.